aviary/validation_cases/benchmark_tests/test_FLOPS_based_sizing_N3CC.py

```diff
@@ -9,557 +9,68 @@
-
-    prob.set_val(
-        'traj.climb.controls:altitude',
-        climb.interp(Dynamic.Mission.ALTITUDE, ys=[alt_i_climb, alt_f_climb]),
-        units='m',
-    )
-    prob.set_val(
-        'traj.climb.controls:mach',
-        climb.interp(Dynamic.Atmosphere.MACH, ys=[mach_i_climb, mach_f_climb]),
-        units='unitless',
-    )
-    prob.set_val(
-        'traj.climb.states:mass',
-        climb.interp(Dynamic.Vehicle.MASS, ys=[mass_i_climb, mass_f_climb]),
-        units='kg',
-    )
-    prob.set_val(
-        'traj.climb.states:distance',
-        climb.interp(Dynamic.Mission.DISTANCE, ys=[distance_i_climb, distance_f_climb]),
-        units='m',
-    )
-
-    prob.set_val('traj.cruise.t_initial', t_i_cruise, units='s')
-    prob.set_val('traj.cruise.t_duration', t_duration_cruise, units='s')
-
-    if use_new_dymos_syntax:
-        controls_str = 'controls'
-    else:
-        controls_str = 'polynomial_controls'
-
-    prob.set_val(
-        f'traj.cruise.{controls_str}:altitude',
-        cruise.interp(Dynamic.Mission.ALTITUDE, ys=[alt_i_cruise, alt_f_cruise]),
-        units='m',
-    )
-    prob.set_val(
-        f'traj.cruise.{controls_str}:mach',
-        cruise.interp(Dynamic.Atmosphere.MACH, ys=[cruise_mach, cruise_mach]),
-        units='unitless',
-    )
-    prob.set_val(
-        'traj.cruise.states:mass',
-        cruise.interp(Dynamic.Vehicle.MASS, ys=[mass_i_cruise, mass_f_cruise]),
-        units='kg',
-    )
-    prob.set_val(
-        'traj.cruise.states:distance',
-        cruise.interp(Dynamic.Mission.DISTANCE, ys=[distance_i_cruise, distance_f_cruise]),
-        units='m',
-    )
-
-    prob.set_val('traj.descent.t_initial', t_i_descent, units='s')
-    prob.set_val('traj.descent.t_duration', t_duration_descent, units='s')
-
-    prob.set_val(
-        'traj.descent.controls:altitude',
-        descent.interp(Dynamic.Mission.ALTITUDE, ys=[alt_i_descent, alt_f_descent]),
-        units='m',
-    )
-    prob.set_val(
-        'traj.descent.controls:mach',
-        descent.interp(Dynamic.Atmosphere.MACH, ys=[mach_i_descent, mach_f_descent]),
-        units='unitless',
-    )
-    prob.set_val(
-        'traj.descent.states:mass',
-        descent.interp(Dynamic.Vehicle.MASS, ys=[mass_i_descent, mass_f_descent]),
-        units='kg',
-    )
-    prob.set_val(
-        'traj.descent.states:distance',
-        descent.interp(Dynamic.Mission.DISTANCE, ys=[distance_i_descent, distance_f_descent]),
-        units='m',
-    )
-
-    # Turn off solver printing so that the SNOPT output is readable.
-    prob.set_solver_print(level=0)
-
-    dm.run_problem(
-        prob,
-        simulate=sim,
-        make_plots=False,
-        simulate_kwargs={'times_per_seg': 100, 'atol': 1e-9, 'rtol': 1e-9},
-        solution_record_file='N3CC_sizing.db',
-    )
-    # prob.run_model()
-    # z=prob.check_totals(method='cs', step=2e-40, compact_print=False)
-    # exit()
-    prob.record('final')
-    prob.cleanup()
+    om.n2(prob)
```

Comment:
Since this is a test, I think we should remove the n2 so that it doesn't pop up when it is run.